docs/source/concept_models/vivarium_swissre_breastcancer/concept_model.rst

```diff
@@ -523,13 +519,13 @@ We assume family history does not affect incidence rates from LCIS to breast can
 
 (1) :math:`RR_{tx|DCIS}` = 0.53 for mastectomy versus no surgery, 0.40 for lumpectomy + radiotherapy versus no surgery, and 0.69 for lumpectomy only versus no surgery, respectively. (Mannu 2020, BMJ)
 (2) PAF = :math:`\frac{P_{tx1}(RR_{tx|DCIS}-1)}{1+P_{tx1}(RR_{tx|DCIS}-1)}`
-(3) :math:`P_{tx1}` = 30% as baseline screening uptake x 100% as treatment uptake
+(3) :math:`P_{tx1}` = 30% as baseline screening uptake x 97.5% as treatment uptake
```

Comment:
Should we include the UI here 97.5% (UI 95-100%)? I see it included in the table below, but might be good to include here as well?